doc/source/release-notes/Percona-Server-5.6.39-83.1.rst

```diff
@@ -0,0 +1,69 @@
+.. rn:: 5.6.39-83.1
+
+=============================
+|Percona Server| 5.6.39-83.1
+=============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.39-83.1 on
+February 12, 2018. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.39-83.1/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.6.39
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-39.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.39-83.1 is now the current
+stable release in the 5.6 series. All of |Percona|'s software is open-source
+and free.
+
+Bugs Fixed
+==========
+
+With :variable:`innodb_large_prefix` set to ``1``, Blackhole storage engine
+was incompatible with InnoDB table definitions, thus adding new indexes would
+cause replication errors on the slave. Bug fixed :psbug:`1126` (upstream
+:mysqlbug:`53588`).
+
+Intermediary slave with Blackhole storage engine couldn't record updates
+from master to its own binary log in case master has
+:variable:`binlog_rows_query_log_events` option enabled. Bug fixed
+:psbug:`1119` (upstream :mysqlbug:`88057`).
+
+A build error on FreeBSD was fixed (:psbug:`2284`).
+
+Inserting special characters into a table caused the audit_log plugin to
```

Comment:
More along the lines of "Server queries that contained JSON special characters and were logged by audit_log (link?) plugin in JSON format caused invalid output due to lack of escaping"